src/components/SVGViewer/stories/full.md

```diff
@@ -0,0 +1,96 @@
+# SVG viewer
+
+<!-- STORY -->
+
+#### Description:
+
+SVG-viewer is a component which visualise `.svg`-documents in a user interactive way.
+
+SVG-viewer provides the following functionality:
+- Display documents with zoom and panning functionality for both desktop and mobile
+- Highlight equipment and provide callbacks for further interaction
+- Locate and zoom onto equipment in the document
+- Search by equipment with locating and zooming results
+
+#### Usage:
+
+```typescript jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { SVGViewer } from '@cognite/gearbox';
+
+function ExampleComponent(props) {
+
+  return (
+    <div style={{ height: '100vh', position: 'relative' }}>
+      <SVGViewer
+        documentId={5185355395511590}
+      />
+    </div>
+  );
+
+}
+```
+
+#### Available props:
+##### Required:
+
+| Property              | Description                                 | Type                        | Default |
+| --------------------- | ------------------------------------------- | --------------------------- | ------- |
+| `documentId`            | CDF fileId to fetch svg-document            | `number` |         |
+
+#### Optional:
+
+| Property              | Description                                 | Type                        | Default |
+| --------------------- | ------------------------------------------- | --------------------------- | ------- |
+| `metadataClassesConditions`            | List of classes and conditions on when they should be applied for equipment            | `Conditions[]` |         |
+| `title` | Document title  | `string`                  |   |
+| `description` | Document description  | `string`                  |   |
+| `showOverlappedText` | Display text with stroke-width: 0  | `string`                  | true |
```

Comment:
Why type is `string` and the default value is `true`?

Reply:
fixed